sycl/test-e2e/ESIMD/unified_memory_api/Inputs/gather.hpp

```diff
@@ -286,3 +468,61 @@ template <typename T, TestFeatures Features> bool testUSM(queue Q) {
   }
   return Passed;
 }
+
+template <typename T, TestFeatures Features> bool testACC(queue Q) {
+  constexpr bool UseMask = true;
+  constexpr bool UsePassThru = true;
+  constexpr bool UseProperties = true;
+
+  properties AlignElemProps{alignment<sizeof(T)>};
+
+  bool Passed = true;
+  Passed &= testACC<T, 1, 1, !UseMask, !UsePassThru, !UseProperties>(
+      Q, 2, AlignElemProps);
+#ifdef __ESIMD_FORCE_STATELESS_MEM
+  Passed &= testACC<T, 2, 1, UseMask, !UsePassThru, !UseProperties>(
+      Q, 2, AlignElemProps);
+  Passed &= testACC<T, 4, 1, UseMask, !UsePassThru, !UseProperties>(
+      Q, 2, AlignElemProps);
+#endif // __ESIMD_FORCE_STATELESS_MEM
+  Passed &= testACC<T, 8, 1, UseMask, !UsePassThru, !UseProperties>(
+      Q, 3, AlignElemProps);
+  Passed &= testACC<T, 16, 1, UseMask, !UsePassThru, UseProperties>(
+      Q, 2, AlignElemProps);
+  Passed &= testACC<T, 32, 1, UseMask, !UsePassThru, !UseProperties>(
+      Q, 3, AlignElemProps);
+
+  if constexpr (Features == TestFeatures::PVC ||
+                Features == TestFeatures::DG2) {
+    properties LSCProps{cache_hint_L1<cache_hint::streaming>,
+                        cache_hint_L2<cache_hint::cached>,
+                        alignment<sizeof(T)>};
+    Passed &=
+        testACC<T, 1, 1, !UseMask, !UsePassThru, UseProperties>(Q, 2, LSCProps);
+    Passed &=
+        testACC<T, 2, 1, UseMask, !UsePassThru, UseProperties>(Q, 2, LSCProps);
+    Passed &=
+        testACC<T, 4, 1, UseMask, UsePassThru, UseProperties>(Q, 2, LSCProps);
+    Passed &=
+        testACC<T, 8, 1, UseMask, UsePassThru, UseProperties>(Q, 3, LSCProps);
+
+    Passed &=
+        testACC<T, 32, 1, UseMask, UsePassThru, UseProperties>(Q, 2, LSCProps);
+
+    // Check VS > 1. GPU supports only dwords and qwords in this mode.
+    if constexpr (sizeof(T) >= 4) {
+      // TODO: This test case causes flaky fail. Enable it after the issue
+      // in GPU driver is fixed.
```

Comment:
Do we have an internal tracker for this or a version we know works?

Reply:
Yes, I created a tracker before uploading this PR.